src/library/previewbuttondelegate.cpp

```diff
@@ -26,6 +25,23 @@ inline TrackModel* trackModel(QTableView* pTableView) {
 
 } // namespace
 
+LibraryPreviewButton::LibraryPreviewButton(QWidget* parent)
+        : QPushButton(parent) {
+    setObjectName("LibraryPreviewButton");
+}
+
+void LibraryPreviewButton::paint(QPainter* painter) {
+    // This matches the implementation of QPushButton::paintEvent, except it
+    // does not create a new QStylePainter, and it is simpler and more
+    // direct than QWidget::render(QPainter*, ...).
+    QStyleOptionButton option;
+    initStyleOption(&option);
+    auto pStyle = style();
```

Comment:
'auto pStyle' can be declared as 'auto *pStyle' [readability-qualified-auto]
```suggestion
    auto* pStyle = style();
```

Reply:
done

Reply:
I removed another unneeded m_TableView member, it's in the base class already.